email-template/src/main/java/io/micronaut/email/template/DefaultEmailTemplateSender.java

```diff
@@ -77,7 +78,10 @@ public void send(@NonNull @NotNull @Valid Sender sender,
     }
 
     @NonNull
-    private Optional<String> content(@NonNull ModelAndView<T> modelAndView) {
+    private Optional<String> content(@Nullable ModelAndView<T> modelAndView) {
+        if (modelAndView == null) {
+            return Optional.empty();
+        }
         if (modelAndView.getView().isPresent()) {
             String viewName = modelAndView.getView().get();
```

Comment:
Maybe you can group both `ifs`: `if (modelAndView != null && modelAndView.getView().isPresent())`